Bottom: 726ce996d2d1a5baf41e95aea6206460f7025ef7
Top:    125554191cd0b1b902213358e1baa2eccdc130e8
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-17 20:32:04 +0000

Refresh of polygon1-c-remove-now-unused

---

diff --git a/src/polygon1.c b/src/polygon1.c
index fff6a7a..a5d1325 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -104,10 +104,6 @@ int vect_inters2 (Vector A, Vector B, Vector C, Vector D, Vector S1,
 
 #define error(code)  longjmp(*(e), code)
 
-#define MemGet(ptr, type) \
-  if (UNLIKELY (((ptr) = (type *)malloc(sizeof(type))) == NULL))	\
-    error(err_no_memory);
-
 #undef DEBUG_LABEL
 #undef DEBUG_ALL_LABELS
 #undef DEBUG_JUMP
